apps/gbfs/lib/gbfs/controllers/smoove_controller.ex

```diff
@@ -4,127 +4,141 @@ defmodule GBFS.SmooveController do
   require Logger
 
   plug(:put_view, GBFS.FeedView)
+  @gbfs_version "1.1"
 
   @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
   def index(conn, _params) do
     contract_id = conn.assigns.smoove_params.contract_id
 
-    conn
-    |> assign(
-      :data,
-      %{
-        "fr" => %{
-          "feeds" =>
-            Enum.map(
-              [:system_information, :station_information, :station_status],
-              fn a ->
-                %{
-                  "name" => Atom.to_string(a),
-                  "url" => apply(Routes, String.to_atom("#{contract_id}_url"), [conn, a])
-                }
-              end
-            )
-        }
-      }
-    )
-    |> render("gbfs.json")
+    {:ok,
+     %{
+       "fr" => %{
+         "feeds" =>
+           Enum.map(
+             [:system_information, :station_information, :station_status],
+             fn a ->
+               %{
+                 "name" => Atom.to_string(a),
+                 "url" => apply(Routes, String.to_atom("#{contract_id}_url"), [conn, a])
```

Comment:
Pour info on peut avoir à faire attention avec les conversions vers les atoms, si la donnée d'input est arbitraire / extérieure (https://til.hashrocket.com/posts/gkwwfy9xvw-converting-strings-to-atoms-safely).

Là dans ce cas ma compréhension est que la liste des contract_id est fixe et contrôlée par nous, donc pas de problème.